x/slashing/internal/types/params.go

```diff
@@ -164,3 +170,19 @@ func validateSlashFractionDowntime(i interface{}) error {
 
 	return nil
 }
+
+func validateSlashFractionBeaconInactivity(i interface{}) error {
```

Comment:
I think there might need to be a check that i is not nil - for the minSelfDelegation validate it could be nil sometimes which would panic

Reply:
Yes, thanks.